### PR TITLE
fix(TYP-2386): Pass hidden fields via fragment

### PR DESCRIPTION
**Pull request by mathio**
Report: [TYP-2386](https://jira.typeform.tf/browse/TYP-2386)

Bug introduced in [DIST-241](https://jira.typeform.tf/browse/DIST-241) by changing passing of hidden fields from query param to fragment.

### CODE REVIEW

File: demo/demo.js [hunk 1]
```diff
@@ -7,36 +7,35 @@ window.addEventListener('message', function (event) {
...
+//   EMBED_DOM_CLASSES
+//     .forEach(function (embed) {
+//       var attribute = embed.attribute
+//       var embeds = document.querySelectorAll(embed.selector)
+//       embeds.forEach(function (embed) {
+//         var embedAttr = embed.getAttribute(attribute)
+//         embed.setAttribute(attribute, addQueryStringToUrl(embedAttr, queryString))
+//       })
+//     })
+// })
```
Comment (prilutskiy) [inline demo/demo.js:41]: Does it make sense to remove commented lines entirely?

Comment (mathio) [reply demo/demo.js:41]: Yes, I will remove it!

**Comment by travis-tf** [comment]
*[BOT]* Preview available: https://embed.typeform.com/5c3b9aa9ee623fc101b5c3cad7c0c4ceeac74f77/embed.js

**Pull request merged**